### ISSUE TITLE
Compatibility with https://github.com/overhangio/openedx-scorm-xblock/ ?

### ISSUE THREAD

**Issue by regisb** [Abstract-Tech/abstract-scorm-xblock#4]
Hi!

It appears that we are developing different XBlocks from the same original fork :-/ I have very little interest in competing with your XBlock, and I think it would make sense to coordinate to avoid effort duplication. This makes more sense now that I've added the custom storage backend feature: https://github.com/overhangio/openedx-scorm-xblock/#custom-storage-backends If I understand correctly, this would allow you to store the xblock assets in MongoDb, as you do now with your xblock.

What do you think?

(I tried to get in touch privately a couple weeks ago via Slack but did not get an answer)

**Comment by silviot** [comment]
Hi Regis!
We are indeed developing two forks originating from the same xblock.
We needed a SCORM xblock that supported import/export of courses; we figured one way to do this would have been to use the "files and uploads" section of the course to upload the SCORM zip file; our xblock does not take care of the upload, but only takes a reference to an existing file.
We worked on top of the existing xblock, changing its structure to achieve our goal, and then released it.
We named it `abstract-scorm-xblock` because it's the one we developed and the one we install for our customers.
It doesn't sound to us we're duplicating efforts: our goals are different.
